### ISSUE TITLE
Cannot change workspace for a time entry [MAC]

### ISSUE THREAD

**Issue by RockyMM** [toggl/toggldesktop#2312]
I entered a worklog with description but I noticed that the project that I needed was not in the list. Actually, when I looked a bit better, only one workspace was in the list and I have four.

Am I missing something or it looks like Toggle desktop supports only single workspace?

**Comment by IndrekV** [comment]
Toggl desktop should show all the same data that is available in the web app.  This might be a syncing issue. Please try to logout of Toggl Desktop and then log back in again. This triggers full data sync and should get all data from the servers.

If this still does not help, please send feedback from the app so we get the log files and can investigate.

**Comment by tonin** [comment]
Hi @RockyMM !  You, here !? ;-)  I had the same behaviour and indeed logging out and then in made the other projects appear for me.

But @IndrekV, can't the sync process be improved so that this doesn't happen?  I've seen that a few times, although I don't have a way to reproduce it.  Or can a menu item or a box in the preferences pane be added to force the full sync?  Thanks!

**Comment by IndrekV** [comment]
I agree that there should be a reasonably easy way to trigger full sync if needed. The functionality is already present. We just need to figure out where the button or menu item fits the best. Or maybe just make every other sync a full sync. If user triggers sync twice one of the syncs is full sync.

**Comment by RockyMM** [comment]
@tonin  yep, discovered Toggle, it's really cool. 👍 

Unfortunately, I had to switch to Windows on my work, and I forgot to test this. Maybe over weekend I can check workspaces in my Mac app.

**Comment by kurko** [comment]
FYI, clicking the `sync` menu item didn't solve the issue for me, had to logout, close app, reopen.

**Comment by thisisabhinay** [comment]
Have same issue. Can't change workspace in macOS app
